CIScripts/Common/Invoke-NativeCommand.ps1

```diff
@@ -33,6 +33,4 @@ function Invoke-NativeCommand {
     if ($AllowNonZero) {
         Write-Output $LastExitCode
     }
-
-    $Global:LastExitCode = $null
```

Comment:
I guess this is not needed, because now the `$Error` [is checked before](https://github.com/jenkinsci/durable-task-plugin/blob/4a06f3cc8eb9053c0bf5597a06b69163c005c3b0/src/main/java/org/jenkinsci/plugins/durabletask/PowershellScript.java#L103) `$LastExitCode`?

Anyway, I'd keep this line, so we don't lose compatibility with currently used version of durable task plugin.

Reply:
It passed CI, so it is compatible with current version of durable-task-plugin

Reply:
You're right, it passed CI, but unfortunatelly it detects less errors. With this line removed the following errors won't be catched:
1. Any error thrown after successful `Invoke-NativeCommand` (only with current version of durable task plugin)
2. Errors like "Command not found" (also with new version)